packages/apps/plugins/plugin-map/src/components/MapControl.tsx

```diff
@@ -21,7 +21,7 @@ export type Marker = {
 
 const defaults = {
   center: { lat: 37.970833, lng: 23.72611 } as LatLngExpression,
-  zoom: 13,
+  zoom: 1,
```

Comment:
Zoom way out to guarantee including the pins. A future improvement would be to set zoom and pan to center the pins.

Reply:
https://github.com/PaulLeCam/react-leaflet/issues/190 Looks like it might be somewhat trivial to achieve this?

Reply:
this seems like it would make this much more useful

Reply:
Agreed, I'll take care of it. I didn't want to put much polish into the PR until I knew it was overall on the right track.

Reply:
Have something for it here: https://github.com/dxos/dxos/pull/7583/commits/e24455c8763b80ee5c7196443eb77ef13c64b21b

Reply:
Thank you for the `latLngBounds` tip!